### DISCUSSION TITLE
Character Bounding Boxes

### DISCUSSION THREAD

**Discussion by RangeOfGlitching** [PaddlePaddle/PaddleOCR#7053] [Q&A] [answered]
Hello,
I have a question. Is that possible to bounding each character insted of whole vocabulary?
"apple"
"a" "p" "p" "l" "e"

I use Easyocr before, but It seem like It can't  make it.
https://github.com/JaidedAI/EasyOCR/issues/631?fbclid=IwAR1yfDmdDvTfxSlchqpC45KMy6Y3KnBju9GygOG3RSb5cSLa-luHB0hLSbM

**Comment by tink2123** [comment, marked as answer]
Sorry, the current model does not support the detection of a single character. What scenarios need to identify characters separately?

**Reply by RangeOfGlitching** [reply]
Because I want to double check the character.

**Comment by malikwaheed10** [comment]
It should give bounding box of each character with confidence level.

**Reply by skwskwskwskw** [reply]
Like how?

**Comment by SKRiyaz03** [comment]
Is there any update on this feature #7053

**Comment by iCounterBOX** [comment]
i also look for this feature. Paddle works perfectly on all of my texts ..detection very good on all the vocabulary..What i need is the exact bounding-box position for each letter. I will crop ( ROI ) this letter for further processing ( e.g. as training-material for tensorFlow..).
For sure this is possible also with openCV tricks.. But  as the detection results from paddle are so exact and perfect the idea to "ROI" in this way has some charm i guess

**Reply by hieuhthh** [reply]
any update yet?
